pkgs/jovian-greeter/greeter.py

```diff
@@ -181,8 +185,17 @@ def _find_sessions(self, sessions: Iterable[str]) -> Optional[Session]:
 
         return None
 
+def handle_exception(exc_type, exc_value, exc_traceback):
```

Comment:
Don't those get logged to greetd journal? I thought they did? Also, should probably exit(1) after.